src/client/components/pages/revision.jsx

```diff
@@ -117,6 +117,9 @@ const EntityLink = require('../entity-link.jsx');
 				</div>
 			));
 
+			const editorTitle =
```

Comment:
This is slightly problematic - it will soon be possible for editors to reply to notes on revisions, so there will be notes by several different editors. So the title should be retrieved on a note-by-note basis. If there's a `title` relation on the editor data model, then that should be fairly easy to do by modifying the server code that fetches the note author to also fetch `author.title`.